frontend/src/templates/Field/ChildrenCompound/ChildrenCompoundField.tsx

```diff
@@ -324,12 +326,24 @@ const ChildrenBody = ({
           const key = `${field.id}+${index}`
           const fieldPath = `${schema._id}.child.${currChildBodyIdx}.${index}`
           const myInfoValue = getChildAttr(subField)
+          let myInfoFormattedValue
```

Comment:
nit but I'm thinking it might be better to use const here instead

```
const myInfoFormattedValue = subField === MyInfoChildAttributes.ChildDateOfBirth && myInfoValue 
? formatMyinfoDate(myInfoValue)
: myInfoValue
```

Reply:
yup good point, done in [ee15c01](https://github.com/opengovsg/FormSG/pull/6648/commits/ee15c01c43f6f7661d89407a7602f2910a10f7c5)